.cirrus.yml

```diff
@@ -558,7 +558,7 @@ apiv2_test_task:
         $CIRRUS_PR == '' ||
         $CIRRUS_CHANGE_TITLE =~ '.*CI:ALL.*' ||
         changesInclude('.cirrus.yml', 'Makefile', 'contrib/cirrus/**', 'vendor/**', 'test/tools/**', 'hack/**', 'version/rawversion/*') ||
-        changesInclude('test/apiv2/**') ||
+        changesInclude('test/apiv2/**', 'test/python/**', 'test/registries.conf', 'test/registries-cached.conf') ||
```

Comment:
It is not wrong to add `'test/registries.conf', 'test/registries-cached.conf'`, however this effects basically all other tasks too. I think they must go into the danger zone condition instead

Reply:
Good point.  I'll make the change and update the danger-zone test as well.